requirements/pip.txt

```diff
@@ -31,15 +31,9 @@ pyyaml==5.3.1
 Pygments==2.7.3
 
 # Basic tools
-# Redis 3.x has an incompatible change and fails
-# https://stackoverflow.com/questions/53331405/django-compress-error-invalid-input-of-type-cachekey
-# https://github.com/sebleier/django-redis-cache/pull/162
```

Comment:
I was able to run `collectstatic` without problem and I haven't seen any obvious error for now.